pgscatalog_utils/ancestry/ancestry_analysis.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import textwrap
+import logging
+import glob
+import os
+
+import pandas as pd
+
+import pgscatalog_utils.config as config
+from pgscatalog_utils.ancestry.read import read_pcs, read_pgs, extract_ref_psam_cols
+from pgscatalog_utils.ancestry.tools import assign_ancestry, _assign_method_threshold, choose_pval_threshold, \
```

Comment:
function names that start with `_` are normally for internal use (i.e. inside `pgscatalog_utils.ancestry.tools`)

Reply:
That makes sense, happy to change it